test/Cli/Func.E2E.Tests/Commands/FuncInit/NodeInitTests.cs

```diff
@@ -0,0 +1,186 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Azure.Functions.Cli.E2E.Tests.Traits;
+using Azure.Functions.Cli.TestFramework.Assertions;
+using Azure.Functions.Cli.TestFramework.Commands;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Azure.Functions.Cli.E2E.Tests.Commands.FuncInit
+{
+    [Trait(WorkerRuntimeTraits.WorkerRuntime, WorkerRuntimeTraits.Node)]
+    public class NodeInitTests(ITestOutputHelper log) : BaseE2ETests(log)
+    {
+        [Theory]
+        [InlineData("")]
+        [InlineData("v3")]
+        [InlineData("v4")]
+        public void Init_Node_App_With_SupportedModel(string programmingModel)
+        {
+            var workingDir = WorkingDirectory;
+            var programmingModelFlag = string.IsNullOrEmpty(programmingModel) ? string.Empty : $"--model {programmingModel}";
+            var testName = nameof(Init_Node_App_With_SupportedModel);
+            var funcInitCommand = new FuncInitCommand(FuncPath, testName, Log ?? throw new ArgumentNullException(nameof(Log)));
+            var localSettingsPath = Path.Combine(workingDir, "local.settings.json");
+            var expectedcontent = new[] { "FUNCTIONS_WORKER_RUNTIME", "node" };
```

Comment:
We should use the existing constants for valid strings e.g.

FUNCTIONS_WORKER_RUNTIME -> https://github.com/Azure/azure-functions-core-tools/blob/0e387d75242b291b436027f1115f894bc9f369be/src/Cli/func/Common/Constants.cs#L22


local settings json -> https://github.com/Azure/azure-functions-core-tools/blob/0e387d75242b291b436027f1115f894bc9f369be/src/Cli/func/Common/Constants.cs#L94